scalafix-tests/unit/src/test/scala/scalafix/tests/interfaces/ScalafixArgumentsSuite.scala

```diff
@@ -214,6 +213,74 @@ class ScalafixArgumentsSuite extends AnyFunSuite with DiffAssertions {
     assert(contentAfterRun == fileEvaluation.previewPatches().get)
   }
 
+  test("ScalafixArguments.evaluate retrieve only atomic patches") {
```

Comment:
I think tests would be more readable (and avoid 1/2 suffixes in variables) if you had more tests with less assertions

Reply:
ok